### PR TITLE
Unittest cleanup

### PR DESCRIPTION
**Pull request by dsuponitskiy**

### CODE REVIEW

File: src/pke/unittest/UnitTestPRE.cpp [hunk 1]
```diff
@@ -165,46 +159,48 @@ class ReEncrypt : public ::testing::TestWithParam<TEST_CASE> {
...
-            Ciphertext<Element> reCiphertext4 = cc->ReEncrypt(evalKey, ciphertext4);
+            Ciphertext<Element> reCiphertext4 = cc->ReEncrypt(ciphertext4, evalKey);
             result = cc->Decrypt(newKp.secretKey, reCiphertext4, &plaintextIntNew);
             EXPECT_EQ(plaintextIntNew->GetCoefPackedValue(), plaintextInt->GetCoefPackedValue())
                 << failmsg << " ReEncrypt integer plaintext";
 
             Ciphertext<Element> ciphertext5 = cc->Encrypt(kp.publicKey, plaintextShort);
             Plaintext plaintextShortNew2;
-            Ciphertext<Element> reCiphertext5 = cc->ReEncrypt(evalKey, ciphertext5, kp.publicKey);
+            std::cerr << __FILE__ << ":l." << __LINE__ << std::endl;
```
Comment (jbates-duality) [inline src/pke/unittest/UnitTestPRE.cpp:187]: looks like debug left in.

File: benchmark/src/lib-hexl-benchmark.cpp [hunk 2]
```diff
@@ -39,23 +39,13 @@
 #define _USE_MATH_DEFINES
```
Comment (jbates-duality) [inline benchmark/src/lib-hexl-benchmark.cpp:39]: This file is moving to the openfhe-hexl repo, so we'll need to merge over to there.

File: benchmark/src/Encoding.cpp [hunk 3]
```diff
@@ -32,21 +32,17 @@
 #define _USE_MATH_DEFINES
 #include "benchmark/benchmark.h"
 
-bool runOnlyOnce = true;
+bool runOnlyOnce = true; // TODO (dsuponit): do we need runOnlyOnce???
```
Comment (jbates-duality) [inline benchmark/src/Encoding.cpp:35]: Well, do we?

File: benchmark/src/Encoding.cpp [hunk 4]
```diff
@@ -32,21 +32,17 @@
...
-#include "cryptocontexthelper.h"
-
 #include "encoding/encodings.h"
 #include "lattice/elemparamfactory.h"
-#include "math/hal.h"
 #include "scheme/ckksrns/cryptocontext-ckksrns.h"
 #include "gen-cryptocontext.h"
 
-using namespace std;
+#include <iostream>
```
Comment (jbates-duality) [inline benchmark/src/Encoding.cpp:43]: needed?  here because there was debugging at one point?

File: benchmark/src/lib-hexl-benchmark.cpp [hunk 5]
```diff
@@ -39,23 +39,13 @@
...
-#include "cryptocontextgen.h"
-#include "cryptocontexthelper.h"
-
 #include "utils/debug.h"
-
 #include "lib-hexl-util.h"
-#include "math/hal.h"
 
-using namespace std;
+#include <iostream>
```
Comment (jbates-duality) [inline benchmark/src/lib-hexl-benchmark.cpp:46]: needed?  here because there was some debugging at one time?

File: src/pke/include/palisade.h [hunk 6]
```diff
@@ -51,6 +51,6 @@ using std::string;
 
 #include "ciphertext.h"
 #include "cryptocontext.h"
-#include "cryptocontexthelper.h"
+//#include "cryptocontexthelper.h"
```
Comment (jbates-duality) [inline src/pke/include/palisade.h:54]: commented include.  so remove?

File: src/pke/unittest/utbfvrns/UnitTestBFVrnsSerialize.cpp [hunk 7]
```diff
@@ -33,27 +33,27 @@
...
+    }
 };
 
-CryptoContext<DCRTPoly> GenerateTestDCRTCryptoContext(const string& parmsetName,
-    usint nTower,
-    usint pbits) {
-    CryptoContext<DCRTPoly> cc =
-        CryptoContextHelper::getNewDCRTContext(parmsetName, nTower, pbits);
+CryptoContext<DCRTPoly> GenerateTestDCRTCryptoContext(const string& parmsetName, usint nTower, usint pbits) {
+    // TODO (dsuponit): getNewDCRTContext() should be replaced with a context generator function call
```
Comment (jbates-duality) [inline src/pke/unittest/utbfvrns/UnitTestBFVrnsSerialize.cpp:54]: Still needs doing?

File: src/pke/work-in-progress/unittests/UnitTestENCRYPT.cpp [hunk 8]
```diff
@@ -1,130 +0,0 @@
-//==================================================================================
```
Comment (jbates-duality) [inline src/pke/work-in-progress/unittests/UnitTestENCRYPT.cpp:1]: This file was moved but appears that tracking was lost.

File: src/pke/work-in-progress/unittests/UnitTestEncoding.cpp [hunk 9]
```diff
@@ -1,284 +0,0 @@
-//==================================================================================
```
Comment (jbates-duality) [inline src/pke/work-in-progress/unittests/UnitTestEncoding.cpp:1]: This file was moved, but appears tracking was lost.

File: src/pke/work-in-progress/unittests/utbgvrns/UnitTestBGVrns.cpp [hunk 10]
```diff
@@ -1,1134 +0,0 @@
-//==================================================================================
```
Comment (jbates-duality) [inline src/pke/work-in-progress/unittests/utbgvrns/UnitTestBGVrns.cpp:1]: This file was moved/renamed, but appears tracking was lost.

File: src/pke/work-in-progress/unittests/utbgvrns/UnitTestBGVrnsSerialize.cpp [hunk 11]
```diff
@@ -1,404 +0,0 @@
-//==================================================================================
```
Comment (jbates-duality) [inline src/pke/work-in-progress/unittests/utbgvrns/UnitTestBGVrnsSerialize.cpp:1]: This file was moved/renamed, but appears tracking was lost.

**Review by jbates-duality — COMMENTED** [review]
No inherent issues to my eye.  Some cosmetics / include file / move without tracking issues.  Couple of debug cerr << still in one place.

I'm not going to approve at this time, as I want to let Yuriy have a whack at it.

File: benchmark/src/Encoding.cpp (hunk 3 above)
Comment (dsuponitskiy) [reply benchmark/src/Encoding.cpp:35]: I added comments as I didn't want my finding to be lost. All those things are to be corrected/changed later.

File: src/pke/unittest/utbfvrns/UnitTestBFVrnsSerialize.cpp (hunk 7 above)
Comment (dsuponitskiy) [reply src/pke/unittest/utbfvrns/UnitTestBFVrnsSerialize.cpp:54]: I added comments as I didn't want my finding to be lost. All those things are to be corrected/changed later.

**Review by yspolyakov — APPROVED** [review]

**Pull request merged**